contents/docs/libraries/android/index.mdx

```diff
@@ -181,13 +181,13 @@ Group analytics allows you to associate the events for that person's session wit
 -   Associate the events for this session with a group
 
 ```java
-PostHog.with(this).group("organization", "42dlsfj23f") // organization is the group type, 42dlsfj23f is the group ID
+PostHog.with(this).group("organization", "company_id_in_your_db") // organization is the group type, company_id_in_your_db is the group ID
```

Comment:
In this PR I also decided to update "42dlsfj23f" to "company_id_in_your_db" to make it clearer what it is